vale/styles/spectrocloud/condescending.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
Why do we need the dashes here? The rule seems to be working fine without them 🤔